src/services/feedbackService.ts

```diff
@@ -0,0 +1,33 @@
+import axios from "axios";
+
+const feedbackSubmitEndpoint =
+    "https://docs.google.com/forms/d/e/1FAIpQLSfGc83oJZbPT1f5ghVNYx_qNBczu_803mnlKBloHjRaX3Z6-Q/formResponse?usp=pp_url";
+
+export const submitFeedback = async (feedback: string) => {
+    var body: { [fieldName: string]: string } = {};
+    body["entry.894361764"] = feedback;
+
```

Comment:
This might be something I do, but given we have 2 places with this hack code, may be worth moving this to a single file, and having both of these services leverage this shared code. 

If unclear, you can won’t fix this, but should do sooner than later as well